### ISSUE TITLE
event.source.itemScope is undefined

### ISSUE THREAD

**Issue by noullet** [a5hik/ng-sortable#347]
Hi,

Since 1.3.7, _event.source.itemScope_ is _undefined_ when accessed in _orderChanged_ and _itemMoved_.

The change https://github.com/a5hik/ng-sortable/commit/29b56a14bad699b3007f0f56832677a6fdba3c3a motivated by #93 is the cause of this issue.

In my case, I want to get the model value:

``` javascript
orderChanged: function(event) {
    var model = event.source.itemScope.modelValue;
    // ...
},
```

And here is what I get:

> angular.js:13920 TypeError: Cannot read property 'modelValue' of undefined

I don't think I'm doing anything special but I am surprised that this issue hasn't be raised yet. Please tell me if you can't reproduce it.

**Comment by pwoji** [comment]
This is also happening to me.

**Comment by mariberti84** [comment]
bug exists for me as well.

**Comment by mhmaguire** [comment]
Happening here also

**Comment by pedrosanta** [comment]
+1, Also having this issue. 😕

**Comment by nonplus** [comment]
+1

**Comment by barkerjeff** [comment]
+1, but I have the issue even if going back to 1.3.1...

**Comment by dllabs** [comment]
+1

**Comment by skython** [comment]
+1

**Comment by juanto** [comment]
Happening here too

**Comment by natan88** [comment]
In version 1.3.6 has this error.
In version 1.3.4 does not have this error.

**Comment by flpwgr** [comment]
:+1:

**Comment by a5hik** [comment]
fixed in 1.3.8
